subliminal/providers/legendastv.py

```diff
@@ -345,26 +346,34 @@ def query(self, language, title, season=None, episode=None, year=None):
         # iterate over titles
         for title_id, t in titles.items():
             # discard mismatches on title
```

Comment:
Add something like `logger.info('Evaluating title candidate %d', title_id)` this will avoid repeating it in other logs.